/**:
  ros__parameters:
    time_window: 2.0

    # Parameters for data validation
    distance_travelled_threshold: 100.0
